helloscala-lagom-client/src/main/scala/com/example/client/HelloLagomService.scala

```diff
@@ -0,0 +1,19 @@
+package com.example.client
+
+import com.lightbend.lagom.scaladsl.api.{ Service, ServiceCall, ServiceCallTransport }
+import io.grpc.examples.helloworld.{ HelloReply, HelloRequest }
+
+
+trait HelloLagomService extends Service {
+
+  import Service._
+
+  override def descriptor() = {
+    named("helloworld.Greeter").withCalls(
+      namedCall("SayHello", sayHello _)
+    ).withServiceCallTransports(ServiceCallTransport.Grpc)
+  }
+
+  def sayHello(): ServiceCall[HelloRequest, HelloReply]
+
+}
```

Comment:
Equivalent to @jroper internal demo.
We should stress the importance of `helloworld.Greeter` and `SayHello` so that the appropriate remote endpoint is invoked.

See the generated code in the server end:

<img width="785" alt="screen shot 2017-05-25 at 18 45 39" src="https://cloud.githubusercontent.com/assets/762126/26460591/6d7e4dd6-417a-11e7-80e8-60f1601ad6d7.png">